.github/workflows/release.yml

```diff
@@ -3,6 +3,7 @@ on:
   push:
     branches: [main]
   pull_request:
+  tags: ["v[0-9]+.[0-9]+.[0-9]+"]
   workflow_dispatch:
```

Comment:
I think this isnt working, tags is a key below the push key.

It should be this way:
```diff
on:
   push:
     branches: [main]
+    tags: ["v[0-9]+.[0-9]+.[0-9]+"]
   pull_request:
-  tags: ["v[0-9]+.[0-9]+.[0-9]+"]
   workflow_dispatch:
```

Reply:
Yikes. Fixed on master, thanks!